tests/utils/lora/test_merge_lora.py

```
@@ -1458,6 +1547,208 @@ def test_per_expert_unfused_mismatch_warns(self, tmp_path):
+        from torchao.prototype.mx_formats.nvfp4_tensor import NVFP4Tensor
+
+        from axolotl.cli.utils.lora_merge import (
+            _build_peft_layer_and_get_delta,
+            _dequant_nvfp4,
+            _find_param_wrapper_lora,
+            _requant_by_format,
+            merge_lora_sharded_efficient,
+        )
+
+        torch.manual_seed(0)
+        E, H, I, r, alpha = 4, 64, 16, 4, 8
```

Ruff (0.15.20)

[error] Ambiguous variable name: `I`

(E741)